### ISSUE TITLE
Нормализовать входные изображения

### ISSUE THREAD

**Issue by MisterMap** [MisterMap/undeepvo#18]
Посчитать Mean и std

**Comment by dinarkino** [comment]
Решение: https://discuss.pytorch.org/t/about-normalization-using-pre-trained-vgg16-networks/23560/6
